params/extras/config.go

```diff
@@ -14,6 +14,88 @@ import (
 	ethparams "github.com/ava-labs/libevm/params"
 )
 
+var (
+	TestChainConfig = &ChainConfig{
+		AvalancheContext: AvalancheContext{SnowCtx: utils.TestSnowContext()},
+		NetworkUpgrades: NetworkUpgrades{
+			ApricotPhase1BlockTimestamp:     utils.NewUint64(0),
+			ApricotPhase2BlockTimestamp:     utils.NewUint64(0),
+			ApricotPhase3BlockTimestamp:     utils.NewUint64(0),
+			ApricotPhase4BlockTimestamp:     utils.NewUint64(0),
+			ApricotPhase5BlockTimestamp:     utils.NewUint64(0),
+			ApricotPhasePre6BlockTimestamp:  utils.NewUint64(0),
+			ApricotPhase6BlockTimestamp:     utils.NewUint64(0),
+			ApricotPhasePost6BlockTimestamp: utils.NewUint64(0),
+			BanffBlockTimestamp:             utils.NewUint64(0),
+			CortinaBlockTimestamp:           utils.NewUint64(0),
+			DurangoBlockTimestamp:           utils.NewUint64(0),
+			EtnaTimestamp:                   utils.NewUint64(0),
+		},
+	}
+
+	copyAndSet = func(c *ChainConfig, set func(*ChainConfig)) *ChainConfig {
+		newConfig := *c
+		set(&newConfig)
+		return &newConfig
+	}
```

Comment:
I think this style reads nicer but feel free to put it back

Reply:
Fine with me, I just moved it to a "normal" function instead of being a global var which is a bit strange-ish.